fix: Limitations for custom widgets (#1578)
appsmithorg/appsmith-docs: website/docs/reference/widgets/iframe.md

  <figcaption align = "center"><i>Custom Code Editor</i></figcaption>
</figure>

> ### Limitations for custom widgets:

Reviewer: ```suggestion
### Limitations

> With the Iframe widget, you cannot add widgets that rely on underlying platform capabilities, such as:

Reviewer: ```suggestion
You cannot create custom widgets using Iframe that rely on underlying platform capabilities, such as:
```

> * Widgets behaving like a canvas or parent, enabling the dropping of additional widgets (example, Container).

Reviewer: ```suggestion
* Widgets that act as a canvas or parent for other widgets. Eg: _Container_
* ```

> * Widgets acting as modals or drawers on top of the existing canvas.

Reviewer: ```suggestion
* Widgets that act as a _Modal_ or _Drawer_ on top of the existing canvas.
```

> * Using auto height or responsiveness features within your iframe.

Reviewer: ```suggestion
* Cannot use auto-height or responsiveness features for widgets within the Iframe.
```

> Appsmith currently lacks support for HTML formatting and error parsing. As a result, any HTML or CSS errors in the **srcDoc** field are not identified by Appsmith.

Reviewer: ```suggestion

Appsmith currently does not support HTML formatting and error parsing. As a result, Appsmith cannot identify any HTML or CSS errors in the **srcDoc** property.
```

> Be cautious when handling code in the **srcDoc** field, as errors can be encountered.

Reviewer: ```suggestion
```

> For complex widgets with frequent updates, it is advisable to utilize an external service like CodeSandbox or host your own code. This helps to maintain your solution more effectively.

Reviewer: ```suggestion
- For complex widgets with frequent updates, it is advisable to utilize an external service like CodeSandbox or host your own code to maintain your solution more efficiently.
```
